controllers/flowlogspipeline/flp_objects.go

```diff
@@ -255,10 +258,69 @@ func (b *builder) addTransformStages(lastStage *config.PipelineBuilderStage) {
 	agg := enrichedStage.Aggregate("aggregate", []api.AggregateDefinition{})
 	agg.EncodePrometheus("prometheus", api.PromEncode{
 		Port:   int(b.desired.PrometheusPort),
-		Prefix: "flp_",
+		Prefix: "netobserv_",
 	})
 }
 
+func (b *builder) mergeMetricsConfiguration(pipeline config.PipelineBuilderStage) {
+	// read in the metrics configuration file
+	yFileBytes, err := configflp.FlpMetricsConfig.ReadFile("flowlogs-pipeline-metrics.conf.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("Merge Metrics Configuration: \n%s\n", string(yFileBytes))
+	var metricsConfig struct {
+		Pipeline   []config.Stage
+		Parameters []config.StageParam
+	}
+	err2 := yaml.Unmarshal(yFileBytes, &metricsConfig)
+	if err2 != nil {
+		log.Fatal(err2)
+	}
+
+	// merge the new items into the existing parameters configuration
+	parameters := pipeline.GetStageParams()
+
+	// obtain pointers to various parameters structures:
+	var tn1, tn2 *api.TransformNetwork
+	var ea1, ea2 []api.AggregateDefinition
+	var pe1, pe2 *api.PromEncode
+	var extractIndex int
+
+	for i, p := range parameters {
+		if p.Transform != nil && p.Transform.Network != nil {
+			tn1 = p.Transform.Network
+		}
+		if p.Extract != nil && p.Extract.Aggregates != nil {
+			ea1 = p.Extract.Aggregates
+			extractIndex = i
+		}
+		if p.Encode != nil && p.Encode.Prom != nil {
+			pe1 = p.Encode.Prom
+		}
+	}
+	for _, p := range metricsConfig.Parameters {
+		if p.Transform != nil && p.Transform.Network != nil {
+			tn2 = p.Transform.Network
+		}
+		if p.Extract != nil && p.Extract.Aggregates != nil {
+			ea2 = p.Extract.Aggregates
+		}
+		if p.Encode != nil && p.Encode.Prom != nil {
+			pe2 = p.Encode.Prom
+		}
+	}
+	// combine the rules in both lists
+	rules := append(tn1.Rules, tn2.Rules...)
```

Comment:
After testing, it looks like some parameters are almost duplicate :
```
{
    "input":"SrcAddr",
    "output":"srcK8S",
    "type":"add_kubernetes",
    "parameters":"srcK8S_labels"
}
```
and 
```
{
    "input":"SrcAddr",
    "output":"SrcK8S",
    "type":"add_kubernetes"
}
```
Should we try to deduplicate them or should let it as it is?

Reply:
It doesn't seem good to have duplicates here .. the existing `add_kubernetes` rule should prevail imho

Reply:
We no longer take enrichment from the generated metrics config, so there is now no duplication here. We add to the Operator code all the enrichment we think we will need in the various metrics.